README.md

```diff
@@ -1,21 +1,18 @@
 # Block Site
 
-**Block Site** (formerly known as Stop Social Media) is a simple Chrome extension that improves your productivity by blocking the access to time-consuming websites (like during the working hours) as you specify in **Options**. **Block Site** is by default disabled and doesn't block any website until you say so in **Options**.
+**Block Site** (formerly known as Stop Social Media) is a simple Chrome extension that improves your productivity by blocking access to distracting websites as you specify.
```

Comment:
Should I call websites we block "time-consuming websites" or "distracting websites"?

Reply:
I would suggest "distracting websites". For me, the goal of this extension is not to prevent me from spending a lot of time on the website, but from going there in the first place.

Reply:
Let's call it "distracting websites" then. I agree.

I'll update manifest file (as part of this PR) and repository description (once the PR is merged).